cloud/ovh/ovh_ip_loadbalancing_backend.py

```diff
@@ -0,0 +1,302 @@
+    endpoint:
+        required: true
+        description:
+            - The endpoint to use ( for instance ovh-eu)
+    application_key:
+        required: true
+        description:
+            - The applicationKey to use
+    application_secret:
+        required: true
+        description:
+            - The application secret to use
+    consumer_key:
+        required: true
+        description:
+            - The consumer key to use
+    timeout:
+        required: false
+        type: "int"
+        default: 120
+        description:
+            - The timeout in seconds used to wait for a task to be \
+            completed. Default is 120 seconds.
+
+'''
+
+EXAMPLES = '''
+# Adds or modify the backend '212.1.1.1' to a \
+loadbalancing 'ip-1.1.1.1'
+- ovh_ip_loadbalancing name=ip-1.1.1.1 backend=212.1.1.1 \
+state=present probe=none weight=8 \
+endpoint=ovh-eu application_key=yourkey \
+application_secret=yoursecret consumer_key=yourconsumerkey
+
+# Removes a backend '212.1.1.1' from a loadbalancing \
+'ip-1.1.1.1'
+- ovh_ip_loadbalancing name=ip-1.1.1.1 backend=212.1.1.1
+state=absent endpoint=ovh-eu application_key=yourkey \
+application_secret=yoursecret consumer_key=yourconsumerkey
+'''
+
+RETURN = '''
+'''
+
+import sys
+try:
+    import ovh
+    import ovh.exceptions
+    from ovh.exceptions import APIError
+    HAS_OVH = True
+except ImportError:
+    HAS_OVH = False
+
+def getOvhClient(ansibleModule):
+    endpoint = ansibleModule.params.get('endpoint')
+    application_key = ansibleModule.params.get('application_key')
+    application_secret = ansibleModule.params.get('application_secret')
+    consumer_key = ansibleModule.params.get('consumer_key')
+
+    return ovh.Client(
+        endpoint=endpoint,
+        application_key=application_key,
+        application_secret=application_secret,
+        consumer_key=consumer_key
+    )
+
+
+def waitForNoTask(client, name, timeout):
+    currentTimeout = timeout
+    while len(client.get('/ip/loadBalancing/{}/task'.format(name))) > 0:
+        time.sleep(1)  # Delay for 1 sec
+        currentTimeout -= 1
+        if currentTimeout < 0:
+            return False
+    return True
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            name=dict(required=True),
+            backend=dict(required=True),
+            weight=dict(default=8, type='int'),
+            probe=dict(default='none',
+                       choices=['none', 'http', 'icmp', 'oco']),
+            state=dict(default='present', choices=['present', 'absent']),
+            endpoint=dict(required=True),
+            application_key=dict(required=True, no_log=True),
+            application_secret=dict(required=True, no_log=True),
+            consumer_key=dict(required=True, no_log=True),
+            timeout=dict(default=120, type='int')
+        )
+    )
+
+    if not HAS_OVH:
+        module.fail_json(msg='ovh-api python module\
+        is required to run this module ')
```

Comment:
Using backslashes in single quoted strings like this will have an odd effect.  python won't know quite how much space to embed in the string when you do it this way. It's better to open and close the string on each line.  Since the entire msg is being constructed inside of parenthesis, you don't need the backslashes at all.  Putting those both together, this looks like this:

```
        module.fail_json(msg='ovh-api python module'
            ' is required to run this module')
```